test/common/common/log_verbosity_update_test.cc

```diff
@@ -0,0 +1,229 @@
+#include <string>
+
+#include "source/common/common/fancy_logger.h"
+#include "source/common/common/logger.h"
+
+#include "test/test_common/logging.h"
+
+#include "absl/strings/string_view.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
+TEST(Fancy, safeFileNameMatch) {
```

Comment:
Seems like you have 3 different sections can you comment over them what each section is testing

Reply:
Added comments and re-arranged them.